setup.py

```diff
@@ -0,0 +1,32 @@
+from setuptools import find_packages, setup
+
+import djangocms_fil_bootstrap
+
+
+INSTALL_REQUIREMENTS = [
+    "Django>=1.11,<2.2",
+    # 'django-cms>=4.0',
```

Comment:
we need cms 4, isn't it?

Reply:
Sure, it's not released yet, so there's no point in having that dependency listed